src/crypto/aes.rs

```diff
@@ -175,6 +176,16 @@ pub fn decrypt_user_master_key(
     Ok(fixed_decrypted_master_key)
 }
 
+struct ConstantNonceGenerator {
+    iv: [u8; aead::NONCE_LEN],
+}
+
+impl aead::NonceSequence for ConstantNonceGenerator {
+    fn advance(&mut self) -> Result<aead::Nonce, Unspecified> {
+        Ok(aead::Nonce::assume_unique_for_key(self.iv))
```

Comment:
If we wanted to be a little more explicit, we could keep a flag for if the generator had been previously used and return an `Err` in that case. (assuming my understanding of the usage of this struct is correct)